lnwire/neighbor_ack_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016 Bitfury Group Limited
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php
+
+package lnwire
+
+import (
+       "bytes"
+       "testing"
+
+       "github.com/roasbeef/btcd/wire"
+)
+
+func TestNeighborAckMessageEncodeDecode(t *testing.T) {
```

Comment:
NeighborAckMessage does not contain any information internally so any two NeighborAckMessage are deep-equal. So there is no need to check for equality.